vrk-kpa/suomidigi
# Suopa 311 composer git hooks

TO TEST:
- Do a make fresh or composer install
- Try to commit something (should not trigger tests/linting)
- Try to push something (should trigger tests/linting)
